### DISCUSSION TITLE
本科-三级标题-字号错误

### DISCUSSION THREAD

**Discussion by Carl7yan** [note286/xduts#187] [Q&A] [answered]
大佬，请教一个问题。

本科毕业论文规范中，没有规定三级标题的格式，但是我的论文中需要用到三级标题，我使用了\subsection设置。通过一站式的校内毕业论文格式检测显示，三级标题出现图示字号错误。（图中的字体错误已解决，请忽略）
期待您的回复

（另，非常感谢您创建和维护次模版的付出，BR）

<img width="195" alt="截屏2024-05-01 10 13 21" src="https://github.com/note286/xduts/assets/94885426/675fedd6-fdc9-40af-a306-f7740d02ed02">

**Comment by note286** [comment, marked as answer]
```latex
\documentclass{xduugthesis}
\xdusetup{style/subsec-zihao=-4}
\begin{document}
\end{document}
```

**Reply by Carl7yan** [reply]
感谢！！

**Comment by note286** [comment]
https://github.com/note286/xduts/commit/020d981a1f82b1a810d7a16f108bc935a1e0ef29 已经修复。
